gm4_combat_expanded/data/gm4_combat_expanded/loot_tables/mob/equip_arrow/stray_snowy.json

```diff
@@ -0,0 +1,71 @@
+{
+  "pools": [
+    {
+      "rolls": 1,
+      "entries": [
+        {
+          "type": "minecraft:item",
+          "name": "minecraft:tipped_arrow",
+          "functions": [
+            {
+              "function": "minecraft:set_nbt",
+              "tag": "{CustomPotionEffects:[{Id:2b,Amplifier:2b,Duration:600}],CustomPotionColor:5926017,HideFlags:32}"
```

Comment:
Should've catch this before, but potion effects are now ints and not bytes. 
You can do a find and replace (with regex on) to fix them. Only modify the combat_expanded folder tho (add it to the "files to include")
```
Find:      Id:([\d]+)b
Replace:   Id:$1
```